### DISCUSSION TITLE
Not able to install v8@3.15

### DISCUSSION THREAD

**Discussion by ashwani-zomato** [Homebrew/discussions#2864] [Casks]
Running 
`brew install v8@3.15`

gives this error

Error: v8@3.15: no bottle available!
You can try to install from source with:
  brew install --build-from-source v8@3.15
Please note building from source is unsupported. You will encounter build
failures with some formulae.

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`?

**Reply by ashwani-zomato** [reply]
PS: I am using apple M1 chip

**Reply by carlocab** [reply]
`v8@3.15` does not build on M1 macOS. You can install Rosetta and a `/usr/local` installation of Homebrew if you need `v8@3.15`.

**Comment by ashwani-zomato** [comment]
> brew config

HOMEBREW_VERSION: 3.3.12
ORIGIN: https://github.com/Homebrew/brew
HEAD: 768b172393339b4654107dc6a840a2cd08dc64b9
Last commit: 6 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: caea5aa1341017da56a6584ebc7e7f3e244ca174
Core tap last commit: 3 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 13.0.0 build 1300
Git: 2.32.0 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.1-arm64
CLT: 13.2.0.0.1.1638488800
Xcode: N/A
Rosetta 2: false

**Comment by ashwani-zomato** [comment]
> brew doctor

Your system is ready to brew.
